autrainer/core/scripts/inference_script.py

```diff
@@ -123,7 +124,18 @@ def add_arguments(self) -> None:
                 "Defaults to False."
             ),
         )
-
+        self.parser.add_argument(
+            "-f",
+            "--frequency",
+            type=int,
```

Comment:
You can add bool arguments via:

```python
a = argparse.ArgumentParser()
a.add_argument("-f", "--frequency", default=False, action="store_true")
...
foo = a.parse_args()
```
And then do "foo.f" to access it.

To avoid negative logic, if you really want to make progress bars active by default, I would name this to `"-s", "--silent"`.

Reply:
Sorry, I was too hasty to comment. I see now that it also allows you to specify the number of steps..In that case, it makes sense to make it an `int` of course.

BUT I really don't think it's a good idea to name it `frequency` for an AUDIO toolkit ;)

Reply:
Good point, but we need to find an abbreviation that works, -p is already taken for inference.

Reply:
Changed to `-u / --update-frequency` for preprocessing, inference, and serving in https://github.com/autrainer/autrainer/pull/33/commits/060786dd6b9f75d1a87b969b4ce52a7cfa28c630.